core/platform/models.py

```diff
@@ -113,59 +152,127 @@ def import_email_services(cls):
 
     @classmethod
     def import_memcache_services(cls):
+        """Importing the required memcache services
+
+        Returns:
+            Module gae_memcache_services from core.platform.memcache
+        """
         from core.platform.memcache import gae_memcache_services
         return gae_memcache_services
 
     @classmethod
     def import_taskqueue_services(cls):
+        """Importing the required task queue services
+
+        Returns:
+            Module gae_taskqueue_services from core.platform.taskqueue
+        """
         from core.platform.taskqueue import gae_taskqueue_services
         return gae_taskqueue_services
 
     @classmethod
     def import_search_services(cls):
+        """Importing the required search services
+
+        Returns:
+            Module gae_search_services from core.platform.search
+        """
         from core.platform.search import gae_search_services
         return gae_search_services
 
     NAME = 'gae'
 
 
 class Registry(object):
+    """Model through which _Gae_ model's methods are accessed.
+    This is the model accessed by all submodules
+
+    """
+
+    #mapping methods of this model to the methods of _Gae_ model
     _PLATFORM_MAPPING = {
         _Gae.NAME: _Gae,
     }
 
     @classmethod
     def _get(cls):
+        """Accessor method through which super class's methods can be accessed
+
+        Returns: corresponding method of the super class
+        """
         return cls._PLATFORM_MAPPING.get(feconf.PLATFORM)
 
     @classmethod
     def import_models(cls, model_names):
+        """Importing the required storage models.
+
+        Args:
+            model_names: list(str). List of storage models
+
+        Returns:
+            import_models() of _Gae_ model
+        """
         return cls._get().import_models(model_names)
 
     @classmethod
     def import_current_user_services(cls):
+        """Imports and returns platform-specific current_user_services
+
+        Returns:
+            import_current_user_services() of _Gae_ model
```

Comment:
module. The current_user_services() module.

(Note that this class is meant to be an abstract one, so individual generic methods should not have direct reference to Gae.)